src/pytorch_lightning/loggers/mlflow.py

```diff
@@ -108,6 +113,13 @@ def any_lightning_module_function_or_hook(self):
         save_dir: A path to a local directory where the MLflow runs get saved.
             Defaults to `./mlflow` if `tracking_uri` is not provided.
             Has no effect if `tracking_uri` is provided.
+        log_model: Log checkpoints created by :class:`~pytorch_lightning.callbacks.Checkpoint`
+            as MLFlow artifacts.
+            * if ``log_model == 'all'``, checkpoints are logged during training.
```

Comment:
```suggestion
            as MLFlow artifacts.
            
            * if ``log_model == 'all'``, checkpoints are logged during training.
```
The empty lines are needed so bullet points get rendered correctly in the docs.